uber/models/attendee.py

```diff
@@ -123,9 +123,16 @@ class Attendee(MagModel, TakesPaymentMixin):
 
     badge_printed_name = Column(UnicodeText)
 
-    requested_any_dept = Column(Boolean, default=False)
-
     dept_memberships = relationship('DeptMembership', backref='attendee')
+    dept_membership_requests = relationship(
+        'DeptMembershipRequest', backref='attendee')
+    anywhere_dept_membership_request = relationship(
```

Comment:
Ooh, I like this change! I think it neatly solves the confusion that was in the first pass on this.